### PR TITLE
Load fonts from common services

### PR DESCRIPTION
**Pull request by superdav42**
Updates the PoC to work with fonts from common third party services.

### CODE REVIEW

File: client/checkout/classic/index.js [hunk 1]
```diff
@@ -45,12 +44,29 @@ jQuery( function ( $ ) {
...
+		}
+	}
+
 	const elements = api.getStripe().elements( {
-		fonts: [
-			{
-				cssSrc: upeFontSource,
-			},
-		],
+		fonts: fontRules,
```
Comment (frosso) [inline client/checkout/classic/index.js:69]: (minor) it looks like the approach could be shortened slightly with `document.querySelectorAll`:
```
const fontDomains = [
	'fonts.googleapis.com',
	'fonts.gstatic.com',
	'fast.fonts.com',
	'use.typekit.net',
];

const elements = api.getStripe().elements( {
	fonts: Array.from(
		document.querySelectorAll(
			fontDomains
				.map(
					( domain ) => `link[rel="stylesheet"][href*="${ domain }"]`
				)
				.join( ', ' )
		)
	).map( ( node ) => node.href ),
} );
```

Both approaches seem to be working 👍

**Review by frosso — APPROVED** [review]
Looking good 👍

File: client/checkout/classic/index.js (hunk 1 above)
Comment (superdav42) [reply client/checkout/classic/index.js]: A quick test with console.time() shows my approach is 0.1 ms faster so I'll stick to it

**Comment by superdav42** [comment]
> 	Might be worth considering adding a filter to the fonts array with import { addFilter } from '@wordpress/hooks'; - to allow devs to add/remove their own stylesheets?

Using @wordpress/hooks would introduce additional dependencies on the frontend which will not be used in most cases. Probably better to use php apply_filter() and pass the results to the js code. Should be done in follow up if we are continuing with UPE.

File: client/checkout/classic/index.js (hunk 1 above)
Comment (frosso) [reply client/checkout/classic/index.js:69]: Good thinking on the timing!
In my tests I had an average of 0.048ms difference over 15 tests on Storefront.  But on other themes, the difference was even minor. I'm guessing it depends on how many styles domains there are on the page - the more styles there are, the closer the gap.

**Pull request merged**